src/Validator/PhoneNumber.php

```diff
@@ -70,19 +71,22 @@ final class PhoneNumber extends AbstractValidator
      */
     private ?string $countryContext = null;
 
-    /** @param Options|Traversable<string, mixed>|null $options */
+    /** @param Options|iterable<string, mixed>|null $options */
     public function __construct($options = null)
+    {
+        parent::__construct($options);
```

Comment:
Psalm issue added to baseline stems from this _possibly_ incompatible type. Likely better off addressing this `form` IMO